src/output.c

```diff
@@ -71,8 +71,8 @@ static void *outputThread(void *data) {
    x264_param_default(&params);
    x264_param_default_preset(&params, output->config->outputX264Preset, NULL);
    x264_param_apply_profile(&params, "high");
-   params.i_width = output->config->width;
-   params.i_height = output->config->height;
+   params.i_width = (output->config->width / 2 ) * 2;
+   params.i_height = (output->config->height / 2) * 2;
```

Comment:
Add a comment and fix the formatting error:
```suggestion
   // The resolution has to be even for the encoder to work
   params.i_width = (output->config->width / 2) * 2;
   params.i_height = (output->config->height / 2) * 2;
```